app/models/spree/globalize/translatable.rb

```diff
@@ -10,6 +10,7 @@ module Translatable
 
       class_methods do
         def ransack(params = {}, options = {})
+          params ||= {} 
```

Comment:
params is nil when ajax loading taxon's products in spree backend

Same issue here : https://github.com/spree-contrib/spree_i18n/issues/621